frontend/test/metabase/scenarios/question/filter.cy.spec.js

```diff
@@ -648,6 +648,38 @@ describe("scenarios > question > filter", () => {
     popover().contains(/case/i);
   });
 
+  it("should enable highlighting suggestions with keyboard up and down arrows (metabase#16210)", () => {
+    openReviewsTable({ mode: "notebook" });
+    cy.findByText("Filter").click();
+    cy.findByText("Custom Expression").click();
+
+    cy.get("[contenteditable='true']")
+      .click()
+      .type("c");
+
+    // First highlighted selection is Created At
+    // "C" is wrapped in a <span>
+    // so we must look for string "reated At"
+    cy.findByText("reated At")
+      .closest("li")
+      .should("have.css", "background-color")
+      .and("eq", "rgb(80, 158, 227)");
+
+    cy.get("[contenteditable='true']")
+      .click()
+      .type("{downarrow}");
```

Comment:
I believe you have to tell Cypress not to parse special characters:
https://docs.cypress.io/api/commands/type#Arguments
> To disable parsing special characters sequences, set the `parseSpecialCharSequences` option to false.

Reply:
@nemanjaglumac  Seems to work as it is. Am I missing something?

Reply:
Nothing. I'm just being dumb :D
That's needed when you have `{` in some string. It obviously recognizes their own reserved commands. Ignore me...